.editorconfig

```diff
@@ -7,4 +7,4 @@ indent_style = space
 indent_size = 2
 
 [*.erb]
-indent_size = 4
+indent_size = 2
```

Comment:
yeah I can just kill it, I decided to keep the change in here because it was screwing with my spacing while writing the views.